native/ios/Integreat.xcodeproj/project.pbxproj

```diff
@@ -214,6 +220,11 @@
 		A330D32D22AA9F42005736D0 /* fonts */ = {
 			isa = PBXGroup;
 			children = (
+				2E0440FB26C133EE00183828 /* NotoSans.ttf */,
+				2E0440F426C127AF00183828 /* NotoSans-Bold.ttf */,
+				2E0440F526C127B000183828 /* NotoSans-Regular.ttf */,
```

Comment:
Any reason why we need `NotoSans.ttf` and NotoSans-Regular.ttf`? I think we have just `NotoSans.ttf`.

Reply:
No i think it was not removed properly even i removed it in xcode